src/pages/Home/index.tsx

```diff
@@ -91,41 +97,51 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
 
   return (
     <>
-      {error.status ? (
+      {error.status || milkError.status || toppingsError.status ? (
         <Card>
-          <Card.Body>
-            {error.status} {error.message}
-          </Card.Body>
+          {error ? <Card.Body>{error.message}</Card.Body> : null}
+          {milkError.status ? (
+            <Card.Body>
+              {milkError.status} {milkError.message}
+            </Card.Body>
+          ) : null}
+          {toppings.status ? (
+            <Card.Body>
+              {toppings.status} {toppings.message}
+            </Card.Body>
+          ) : null}
```

Comment:
How did we have extra milk and toppings logic now in the Homepage.
Plus aint you only suppose to be working and finishing boba_list?